google-cloud-firestore/acceptance/firestore/collection_group_test.rb

```diff
@@ -179,5 +179,33 @@
       end
       _(result).must_equal results[1]
     end
+
+    it "queries a collection group using partitions on different timestamps" do
+      rand_col = firestore.col "#{root_path}/query/#{SecureRandom.hex(4)}"
+
+      read_time = Time.now
```

Comment:
Why do you have a sleep before this line in the other tests but not this one?

Reply:
In other tests, sleep was before & after `read_time` because data was being inserted/manipulated. 
Whereas in this test, we are starting with an empty collection -> setting the read_time -> inserting some documents in the collection and then running partition request at 2 different read_time.